### DISCUSSION TITLE
How to handle db migrations of translated source page types?

### DISCUSSION THREAD

**Discussion by leonh** [wagtail/wagtail-localize#465] [Q&A]
I have a site with 1000's of edited translated versions of my ` class LocationPage(Page):` model , about half are just 'alias' version of the source language with the other half being translated either partially or fully.  

If I add a new field to  **LocationPage**  and run my db migrations then I see many warning about the translated page content being out of sync with the source model

_I know this can be fixed on a page-by-page basis through the "sync translated pages" option on the source page._ 

However this will take a very long time to do through the interface....

Is there a _'proper_' way to perform this in bulk , which could be written into a data migration?

Also I have run into error using  "sync translated pages" when the source model has had a field removed?

https://github.com/wagtail/wagtail-localize/blob/8ee52c861b07309a6def36d55efde6d9c67b9ebd/wagtail_localize/segments/ingest.py#L230

Is failing because there is a reference to a field on the source model that no longer exists

I'm not sure how best to handle migrations with regard to my translated content?

**Comment by alice-telescoop** [comment]
This would be very useful ! I don't know if I should reply in here. Should we create an issue for this?
